### PR TITLE
Consistency Refactor

### PR DESCRIPTION
**Pull request by simonbu11**
Refactor to be more consistent with other projects, including:

* Use of app / infrastructure folders
* Use of Jest for testing
* Use of async / await
* Use of lambda over function

### CODE REVIEW

File: src/app/clientManagement/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const session = require('express-session');
-const config = require('./../Config');
+const config = require('./../../infrastructure/Config/index');
```
Comment (dashton82) [inline src/app/clientManagement/index.js]: i think index can be removed

File: src/app/dev/launcher.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-const logoutSource = require('./../logout');
+const logoutSource = require('./../logout/index');
```
Comment (dashton82) [inline src/app/dev/launcher.js]: Remove index?

File: src/app/oidc/getInteraction.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const oidc = require('./oidcServer');
+const config = require('./../../infrastructure/Config');
```
Comment (dashton82) [inline src/app/oidc/getInteraction.js]: does this need to be 
`const config = require('./../../infrastructure/Config')();`

File: src/app/oidc/index.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+const bodyParser = require('body-parser');
+const config = require('./../../infrastructure/Config');
```
Comment (dashton82) [inline src/app/oidc/index.js]: does this need to be 
`const config = require('./../../infrastructure/Config')();`

File: src/app/oidc/oidcServer.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+const uuid = require('uuid/v4');
+const config = require('./../../infrastructure/Config');
```
Comment (dashton82) [inline src/app/oidc/oidcServer.js]: does this need to be 
`const config = require('./../../infrastructure/Config')();`

File: src/infrastructure/adapters/RedisAdapter.js [hunk 6]
```diff
@@ -1,12 +1,12 @@
 /* eslint-disable */
 'use strict';
 const Redis = require('ioredis');
-const epochTime = require('../helpers/epoch_time');
-const config = require('./../Config');
+const epochTime = require('./../helpers/epoch_time');
+const config = require('./../Config/index');
```
Comment (dashton82) [inline src/infrastructure/adapters/RedisAdapter.js]: remove index and does this need to be 
`const config = require('./../../infrastructure/Config')();`

File: src/app/oidc/oidcServer.js (hunk 5 above)
Comment (simonbu11) [reply src/app/oidc/oidcServer.js]: Don't need to do this anymore. Jest.mock with factory removes the problem

File: src/app/oidc/index.js (hunk 4 above)
Comment (simonbu11) [reply src/app/oidc/index.js]: Don't need to do this anymore. Jest.mock with factory removes the problem

File: src/app/oidc/getInteraction.js (hunk 3 above)
Comment (simonbu11) [reply src/app/oidc/getInteraction.js]: Don't need to do this anymore. Jest.mock with factory removes the problem

**Pull request merged**